images/macos/provision/core/powershell.sh

```diff
@@ -35,5 +35,4 @@ sudo ln -s /usr/local/bin/pwsh /usr/local/bin/powershell
 # fix ~/.azure directory permissions
 sudo chown -R ${USER}: $HOME/.azure
 
-# Import Pester Module
-Import-Module Pester
\ No newline at end of file
+pwsh -c "Invoke-PesterTests -TestFile \"Powershell\""
```

Comment:
I don't think it will work. `Invoke-PesterTests` function should be imported before using.
Could you please also wrap it to some bash helper function?